From 1c1cf3becb924748986443ec68722b75346b87ec Mon Sep 17 00:00:00 2001
From: Ken Brown <kbrown@cornell.edu>
Date: Mon, 9 Apr 2018 11:33:55 -0400
Subject: [PATCH] Revert "Disable trapping of floating point exceptions under
 CYGWIN due to strtod bug with real x=121645100408832000.0.""

This reverts commit 2c70c1d965fb0b9fa92f97db2484d3ba4c184f40, which is
no longer needed as of cygwin-2.10.1.
---
 settings.cc | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/settings.cc b/settings.cc
index db3d3963..14336e86 100644
--- a/settings.cc
+++ b/settings.cc
@@ -1355,15 +1355,10 @@ bool view()
 
 bool trap()
 {
-#ifdef __CYGWIN__
-// Disable until broken strtod exception is fixed.
-  return false;
-#else
   if (interact::interactive)
     return !getSetting<bool>("interactiveMask");
   else
     return !getSetting<bool>("batchMask");
-#endif  
 }
 
 string outname() 
-- 
2.16.2

